packages/sdk/src/adapters/multipeer/session.ts

```diff
@@ -168,12 +164,13 @@ export class Session extends EventEmitter {
 		} catch { }
 	}
 
-	private setAuthoritativeClient(clientId: string) {
-		if (!this._clientSet[clientId]) {
+	private setAuthoritativeClient(clientId: Guid) {
+		const newAuthority = this._clientSet.get(clientId);
+		if (!newAuthority) {
 			log.error('network', `[ERROR] setAuthoritativeClient: client ${clientId} does not exist.`);
+			throw new Error(`Client ${clientId} does not exist.`);

```

Comment:
>throw new Error(`Client ${clientId} does not exist.`); [](start = 3, length = 54)

Ensure this exception is handled and doesn't exit SDK code. Or remove it.

Reply:
Would it be better to simply return? I don't think this case is ever hit, because the app would throw later on in this function if it did.

---
In reply to: [372071873](https://github.com/microsoft/mixed-reality-extension-sdk/pull/476#discussion_r372071873) [](ancestors = 372071873)

Reply:
Ya I think just returning would be better. We never hit this that we know of. We don't have a way to recover from this if it were to happen.

---
In reply to: [372075051](https://github.com/microsoft/mixed-reality-extension-sdk/pull/476#discussion_r372075051) [](ancestors = 372075051,372071873)